v20.2/upgrade-cockroach-version.md

```diff
@@ -177,11 +183,9 @@ We recommend creating scripts to perform these steps instead of performing them
     $ systemctl start <systemd config filename>
     ~~~
 
-1. Verify the node has rejoined the cluster through its output to `stdout` or through the [Admin UI](admin-ui-overview.html).
+1. Verify the node has rejoined the cluster through its output to [`stdout`](cockroach-start.html#standard-output) or through the [Admin UI](admin-ui-cluster-overview-page.html#node-status). 
 
-    {{site.data.alerts.callout_info}}
-    To access the Admin UI for a secure cluster, [create a user with a password](create-user.html#create-a-user-with-a-password). Then open a browser and go to `https://<any node's external IP address>:8080`. On accessing the Admin UI, you will see a Login screen, where you will need to enter your username and password.
-    {{site.data.alerts.end}}
+    The node should have at least 70% of the [memory usage](admin-ui-cluster-overview-page.html#node-details) you observed before stopping the node.
```

Comment:
I don't think this advice is sound, it depends too much on how long the node was running previously and what the workload was on the node before it was restarted. For example, if the node was running for long enough that the RocksDB/Pebble cache was full, but there was relatively little load on the cluster, the memory usage on the cluster could be expected to be much lower after starting the node again.